### DISCUSSION TITLE
Set_opacity not working

### DISCUSSION THREAD

**Discussion by arthurmelton** [fltk-rs/fltk-rs#981] [Q&A] [answered]
I have this code 
```rust
    let app = app::App::default();
    let mut wind = Window::default()
        .with_size(950,450)
        .center_screen();
    wind.set_color(Color::from_rgb(25, 25, 25));
    wind.set_opacity(1.0);
    wind.end();
    wind.show();
    while app.wait() {}
```
and it gives this error, I cant tell if I am just dumb or what it looks like it should work

(btw its getting upset at the set opacity)

`assertion failed: 'self.shown()', fltk-1.2.13/src/window.rs:342:9`

**Comment by MoAlyousef** [comment, marked as answer]
Try moving `wind.set_opacity(1.0);` to after `wind.show();`.
